lib/commands/internal.js

```diff
@@ -1,16 +1,35 @@
+const logger = require('../utils/logger');
+
 class InternalCommand {
-	interactive() {}
-	make() {}
-	help() {
-		const HelpGroup = require('../groups/help');
-		const HelpOutput = new HelpGroup().run();
-		console.log(HelpOutput.outputOptions.help);
-	}
-	version() {
-		const jsonPath = require('path').join(__dirname, '..', '..', 'package.json')
-		const version = require(jsonPath, 'utf8').version
-		console.log(`webpack-cli ${version}`);
-	}
+    /**
+     *
+     * @param {String} command
+     * @param {any} args
+     */
+    constructor(command, ...args) {
+        /**
+         *
+         * @type {String}
+         */
+        this.command = command;
+        /**
+         *
+         * @type {any[]}
+         */
+        this.args = args;
+    }
+    interactive() {}
+    make() {}
```

Comment:
you can remove the `make` and `interactive` methods, `make` is not going to be supported, `interactive` is inside `compiler.js` as an `outputOptions` flag